src/main/kotlin/frc/robot/subsystems/wrist/WristConstants.kt

```diff
@@ -13,5 +13,7 @@ enum class Angles(val angle: Angle) {
     L2_ALGAE(Units.Degrees.of(110.0)),
     L3_ALGAE(Units.Degrees.of(110.0)),
     FEEDER(Units.Degrees.of(125.0)),
-    ZERO(Units.Degrees.zero())
+    ZERO(Units.Degrees.zero());
+
+    fun getLoggingName(): String = name.lowercase().replace("_", " ")
```

Comment:
```suggestion
    fun getLoggingName() = name.split("_").joinToString(" ") {
        it.lowercase().replaceFirstChar(Char::uppercase)
    }
```
This will convert `L3_ALGAE` to `L3 Algae`.